### ISSUE TITLE
Skip not working for already downloaded file

### ISSUE THREAD

**Issue by davidchalifoux** [hgouveia/node-downloader-helper#31]
```javascript
    const dl = new DownloaderHelper(fileURL, __dirname, {
      override: {
        skip: true,
        skipSmaller: false,
      },
    })
```

I'm currently using this configuration to skip files that have already been downloaded. According the readme, this should work. However, it is still re-downloading the file. Am I using it wrong?

Note: It is overwriting the old file.

**Comment by hgouveia** [comment]
> ```js
>     const dl = new DownloaderHelper(fileURL, __dirname, {
>       override: {
>         skip: true,
>         skipSmaller: false,
>       },
>     })
> ```
> 
> I'm currently using this configuration to skip files that have already been downloaded. According the readme, this should work. However, it is still re-downloading the file. Am I using it wrong?
> 
> Note: It is overwriting the old file.

Hello @davidchalifoux these changes are not published yet on npm, probably that might be the issue, but you could try installing directly using the github master branch and verify?

`npm i -S hgouveia/node-downloader-helper#master`

if the problem persists, please let me know, and I will look into this

**Comment by davidchalifoux** [comment]
I installed directly from the master branch like you suggested and that fixed it for me.
Thanks for the help, this is a really nice tool!

**Comment by hgouveia** [comment]
Hello @davidchalifoux , i just released 1.0.13 so now you can use the official package from NPM

Thanks for you patience !
